src/judoscale/django/middleware.py

```diff
@@ -28,5 +29,14 @@ def __call__(self, request):
 
             metrics_store.add(metric)
             logger.debug("queue_time={}ms".format(round(queue_time_ms, 2)))
+            self._get_reporter()
```

Comment:
I think I'd like to move this implementation to a public `reporter.ensure_running()` method. We could call the same method from the [Django app initializer](https://github.com/judoscale/judoscale-python/blob/ce0b499333266f317dec0a059229c91a62d9d479/src/judoscale/django/apps.py#L23) rather than calling `reporter.start()` directly.

Does that make sense?

Reply:
Yes, I will replace it. and in that case we need to check if thread is not None before checking its status.